proto/serverless/proto/instrumentation/tagset/v1/tagset.proto

```diff
@@ -0,0 +1,272 @@
+syntax = "proto3";
+
+package serverless.proto.instrumentation.tagset.v1;
```

Comment:
Why it's at `instrumentation/tagset` and not at `trace/tags`?

Are those tags applicable beyond traces?

Reply:
At some point we will apply tags to metrics through a reconciliation process, so these tags are not only for traces.

Reply:
> At some point we will apply tags to metrics through a reconciliation process

Can you elaborate, what would be "reconciliation process" ?

Reply:
That is an ingest issue. We will be supporting defining tags from the Console UI for specific, account ids, orgs, and ARNs. That means as we ingest we will add tags to payloads. As well as potentially take tags from traces as they are ingested and reconciling them into our metric payloads.

Reply:
Ok, so if I understand correctly some tags will also be applied on metrics payloads